<launch>
<param name="/use_sim_time" value="true"/>
<node name="stage" pkg="stage_ros" type="stageros" args="/home/dieisson/physarum_ws/src/physarum_simulation/worlds/map_sonar.world"/>
<!-- Iniciar o map_server com o arquivo map.yaml -->
<node name="map_server" pkg="map_server" type="map_server" args="/home/dieisson/physarum_ws/src/physarum_simulation/worlds/map.yaml" output="screen"/>

 <node name="map_to_odom" pkg="physarum_simulation" type="map_to_odom.py" output="screen"/>
</launch>
